changelog.md

```diff
@@ -2,7 +2,32 @@
 
 ### Master
 
-* Adds `github.api` more to come - @orta
+### 0.18.0
+
+* Adds `github.api`. This is a fully authenticated client from the [github](https://www.npmjs.com/package/github)
+  npm module. - @orta
+
+  An easy example of it's usage would be using Danger to add a label to your PR. Note that Danger will have the 
+  permissions for your account, so for OSS repos - this won't work.
+
+  ```js
+  danger.github.api.issues.addLabels({ owner: "danger", repo: "danger-js", number: danger.github.pr.number, labels: ["Danger Passed"] })
```

Comment:
Hey @orta, quick question: Do we have an API for *receiving* labels as well?

Reply:
Labels are already inside `danger.github.issues.labels` or something like that, http://danger.systems/js/reference.html#GitHubIssue